Author: Matthew John Ernisse <mernisse@ub3rgeek.net>
Description: Replace local version of excanvas.js with libjs-excanvas.
Forwarded: not-needed
Index: mutability-dump1090/public_html/gmap.html
===================================================================
--- mutability-dump1090.orig/public_html/gmap.html
+++ mutability-dump1090/public_html/gmap.html
@@ -13,7 +13,7 @@
 		<script type="text/javascript" src="formatter.js"></script>
 		<script type="text/javascript" src="flags.js"></script>
 		<script type="text/javascript" src="script.js"></script>
-		<script type="text/javascript" src="coolclock/excanvas.js"></script>
+		<script type="text/javascript" src="excanvas.min.js"></script>
 		<script type="text/javascript" src="coolclock/coolclock.js"></script>
 		<script type="text/javascript" src="coolclock/moreskins.js"></script>
         <title>DUMP1090</title>
